### DISCUSSION TITLE
Video tutorials in Spanish,..

### DISCUSSION THREAD

**Discussion by marf-slicer** [lassoan/SlicerBoneReconstructionPlanner#140] [General]
Dear Mauro,

Here, in Canary Islands, as a summer stay of one biomedical eng. student, we did some video tutorials in Spanish of your module.

We did it because it is very useful for our medical students in our Medical School to introduce computer-based tools and for working in clinical cases, and, of course, for biomedical eng. students that are starting their practices.

If you think so, we can share them to you for publishing in this github's page.

Best regards!.

**Comment by mauigna06** [comment]
Dear @marf-slicer 

Thanks for reaching out and your feedback about BRP

> We did some video tutorials in Spanish of your module. We can share them to you for publishing in this github's page.

Yes, that would be very good, but the tutorials need to be created for latest BRP version of Slicer stable release 5.6.2, for that you'll need to wait one or two weeks because I have been making updates to the software and I want to do a thorough test before updating the documentation. 

After those updates, the software will start to be developed differently. Regular changes will be done a "development" branch and the tested branch (I mentioned above) will remain fixed for Slicer 5.6.2, this will avoid introducing unexpected problems/regressions while trying to solve a bug or adding a feature. I'll open an issue and tag you when the tested branch is ready for making the tutorial :)

> We did it because it is very useful for our medical students in our Medical School to introduce computer-based tools and for working in clinical cases, and, of course, for biomedical eng. students that are starting their practices.

It is good to know this, I encourage you to [open a discussion](https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/discussions/new?category=show-and-tell) sharing some cases (virtual surgical plan pictures, OR photos, etc) this helps demonstrating validation of the software which is important for increasing the userbase and can be used for promoting the software

**Comment by marf-slicer** [comment]
Dear Mauro,

Our videos were made under the last version of Slicer (5.6.2).
https://we.tl/t-Q9toJradhZ

Take them, if you consider that they are useful for your github's page 

Best regards!

**Comment by mauigna06** [comment]
Hi
I have downloaded it. I'm on my vacations so I handle this later.
Thanks a lot :)

**Comment by mauigna06** [comment]
I have added the tutorials to the documentation :)
https://github.com/SlicerIGT/SlicerBoneReconstructionPlanner/blob/735c2b1064513fb0ee4e788d9018f7e8ab82dc2b/README.md?plain=1#L197

**Reply by mauigna06** [reply]
Thank you @marf-slicer
